packages/flutter/lib/src/painting/decoration.dart

```diff
@@ -81,6 +81,7 @@ abstract class Decoration {
   /// if it is a [BoxDecoration] with definitely no [DecorationImage]).
   BoxPainter createBoxPainter([VoidCallback onChanged]);
 
+  /// ZZZ this comment needs to be wordsmithed.
```

Comment:
Word smithed by deleting comment and changing this toString method to be a normal toString method that doesn't take any optional arguments. The one case where it looks like this was used was better handled by calling toStringDeep on the decoration instead.